src/components/Asset/AssetSummary/ApplyFormModal/styles.ts

```diff
@@ -55,7 +55,7 @@ export const Content = styled.div<{ opened: boolean }>`
   position: fixed;
   z-index: 7;
 
-  background-color: #181935;
+  background-color: #151515;
```

Comment:
avoid hard coding colors, use the theme file instead.

Reply:
done